src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/conf/WriteYamlStep/config.jelly

```diff
@@ -31,6 +31,9 @@
 	<f:entry title="${%text.title}" field="data" description="${%text.description}">
 		<f:textbox />
 	</f:entry>
+    <f:entry title="${%datas.title}" field="datas" description="${%datas.description}">
+        <f:textbox />
+    </f:entry>
```

Comment:
Have you tested that this works? The datas parameter is likely databound to a string and won't convert to a `Collection` iirc. And also both `data` and `datas` will be specified in the form submission and will generate a code snippet that won't work.

I guess it is better to leave it out?

Reply:
I'm embarrassed to admit I didn't initially know what it was for. I see how it works now, and yeah I don't see a good way to make this work and it's not really necessary either. Removing!